source/_integrations/gardena.markdown

```diff
@@ -0,0 +1,92 @@
+---
+title: "Gardena"
+description: "Instructions on how to setup Gardena devices within Home Assistant."
+logo: gardena.jpg
+ha_category:
+  - Hub
+  - Vacuum
+  - Sensor
+  - Switch
+ha_release: 0.101
+ha_qa_scale: Silver
+ha_config_flow: false
+---
+
+This plugin aims to integrate Gardena smart systems into Home Assistant.
+Gardena provides integrations for mowers, power socket, sensors, ...
+
+[Gardena API](https://developer.1689.cloud/apis).
+[py-smart-gardena wrapper for gardena API](https://github.com/grm/py-smart-gardena).
+
+There is currently support for the following device types within Home Assistant:
+
+- [Mower](#mower)
+- [Sensor](#sensor)
+- [Switch](#switch) (Power plugs, smart water control, Smart irrigation control)
+
+Entities will automatically be added based on what is declared in the Android/IOS 
+application.
+
+Mowers are added as a vacuum.
+
+## Configuration
+
+You first need to get a client id for your personal installation of Gardena. 
+To do so, create an account here: <https://developer.1689.cloud/apis>
+
+Then you need to create an application and get the client_id as explained here: 
+<https://developer.1689.cloud/docs/getting-started>
+
+Once this has been done, you can configure the integration:
+
+```yaml
+# Example configuration.yaml entry
+gardena:
+  email: your_email@provider.com
+  password: your_secret_gardena_password
+  client_id: your_client_id_from_the_gardena_site
+  location_id: your_location_id
+  default_mower_duration_in_minutes: 300
+  default_smart_irrigation_control_duration_in_minutes: 60
+  default_smart_watering_duration_in_minutes: 60
+```
+
+{% configuration %}
+gardena:
+  email: your_email@provider.com
```

Comment:
Unfortunately, that is not how the configuration tag works. Please see the developer documentation for an example and details: <https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

Reply:
My bad, I looked into it too fast. It should be good now.